VMS/Shared/LoginDisplay.razor.scss

```diff
@@ -23,4 +24,156 @@
     @include on('lg') {
         font-size: 70px;
     }
-}
\ No newline at end of file
+}
+
+.login-btn {
+    background-color: $blue;
+    width: 150px;
+    height: 45px;
+
+    &:hover {
+        background-color: $blue-bold;
+    }
+
+    @include on('xl') {
+        width: 200px;
+        height: 60px;
+        font-size: 24px;
+        line-height: 60px;
+    }
+}
+
+.user-section {
+    &__link {
+        /*.user-avatar-wrapper::before {
+            content: "";
+            position: absolute;
+            top: 0;
+            left: 50%;
+            transform: translateX(-50%);
+            width: 80px;
+            height: 100%;
+            border-radius: 50%;
+            background-color: rgba(0, 0, 0, 0.3);
+            z-index: 1;
+            display: none;
+            @include on('xl') {
+                width: 65px;
+            }
+        }*/
```

Comment:
Please delete this comments.

Reply:
I stored these comments here in case we want an overlay on user's avatar when hover on it, but seem like that's unnecessary 🤣 so i've thrown them away